client/src/pages/pot-instance/PotInstance.tsx

```diff
@@ -1,29 +1,59 @@
 import { FC, useState } from 'react';
 import { Button } from '@mui/material';
+import uniqid from 'uniqid';
+import { AddIngredients } from 'pages/pot-instance/components';
 import LandingPage from 'pages/pot-instance/components/LandingPage/LandingPage';
+import PotView, {
+  PotViewState
+} from 'pages/pot-instance/components/PotView/PotView';
 import { PotContent } from 'pages/pot-instance/models';
-import FoodTimerList from './components/FoodTimerList/FoodTimerList';
 import styles from './styles.module.css';
 
 const PotInstance: FC = () => {
   const [potType, usePotType] = useState(0);
-  const [hotpotStart, useHotpotStart] = useState(false);
-  const [cookedPotContent, useCookedPotContent] = useState<PotContent[]>([]);
+  const [hotpotStart, setHotpotStart] = useState(false);
+  const [potContent, setPotContent] = useState<PotContent[]>([]);
+  const [cookedPotContent, setCookedPotContent] = useState<PotContent[]>([]);
 
   // functions to update the landing page
   const updatePotType = (type: number) => {
     usePotType(type);
   };
 
   const startHotPot = () => {
-    useHotpotStart(true);
+    setHotpotStart(true);
   };
 
-  // should we add time when item was completed to this?
-  const addCookedPot = (food: PotContent) => {
-    const tempArray = cookedPotContent;
-    tempArray.push(food);
-    useCookedPotContent([...tempArray]);
+  const endHotPotSession = () => {
+    setHotpotStart(false);
+  };
+
+  // TODO: should we add time when item was completed to this?
+  const addToCookedPot = (foods: PotContent[]) => {
+    setCookedPotContent([...cookedPotContent, ...foods]);
```

Comment:
How does this work? o_o

Reply:
The spread operator just pulls out all of the values in the array. 
So spreading multiple arrays together allows you to form a new array that combines values.